product_packaging_type_required/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Camptocamp (<http://www.camptocamp.com>).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Packaging Type Required",
+    "version": "15.0.1.1.0",
+    "development_status": "Beta",
+    "category": "Product",
+    "summary": "Product Packaging Type Required",
+    "author": "Camptocamp, " "Odoo Community Association (OCA)",
+    "maintainers": ["simahawk", "dcrier"],
+    "website": "https://github.com/OCA/product-attribute",
+    "license": "AGPL-3",
+    "depends": ["product", "product_packaging_type"],
```

Comment:
"product" can be erased as a dependency because "product_packaging_type" also depends on this